donate.html

```diff
@@ -240,9 +240,9 @@ <h2 class="donate-option3-header">$5</h2>
 
   <!-- secondary links -->
   <section class="secondary-links">
-      <a class="nav-link" target="_blank" href="./files/brandingGuide.pdf">Branding</a>
-      <p>•</p>
-      <a class="nav-link" target="_blank" href="./files/terms.pdf">Code of Conduct</a>
+      <a class="nav-link" target="_blank" href="./files/brandingGuide.pdf" style="color: #000 !important; margin-left: 0px !important;">Branding</a>
```

Comment:
sorry heh i wasn't clear in my last comment - this section was fine before! i was more concerned about the little white space on the right of the gradient footer
<img width="475" alt="Screen Shot 2020-06-03 at 1 55 01 PM" src="https://user-images.githubusercontent.com/41929993/83670872-d334ec00-a5a1-11ea-8797-2b52f0a6217a.png">